packages/admin-ui/ui/src/components/fundamentals/icons/tag-red-dot-icon/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import IconProps from "../types/icon-type"
+
+const TagRedDotIcon: React.FC<IconProps> = ({ size = "24", ...attributes }) => {
+  return (
+    <svg
+      width={size}
+      height={size}
+      viewBox="0 0 32 32"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...attributes}
+    >
+      <circle cx="16" cy="16" r="6" fill="#FFE5E5" />
+      <circle cx="16" cy="16" r="3" fill="#E5484D" />
+    </svg>
+  )
+}
+
```

Comment:
suggested(non-blocking): Create a generic `TagStatusDotIcon` with a switch on colors.

```
const TagStatusDotIcon: React.FC<IconProps> = ({ size = "24", color = "some-color", ...attributes }) => {
  let circleOneFill = ""
  let circleTwoFill = ""
  switch (color):
     ...
  return (
```
Though, that would require that all dot icon SVGs be constructed using two circles. I don't know if that's the case. 

Also, this is very nit.